app/mailers/mailer.rb

```diff
@@ -75,6 +75,13 @@ def mfa_recommendation_announcement(user_id)
       subject: "Please enable multi-factor authentication on your RubyGems account"
   end
 
+  def mfa_required_soon_announcement(user_id)
+    @user = User.find(user_id)
+
+    mail to: @user.email,
+      subject: "[Action Required] Enable multi-factor authentication on your RubyGems account by August 15"
```

Comment:
For users with UI only, we should tell them to upgrade their MFA instead of enable it (since it's already enabled)

"[Action Required] Upgrade multi-factor authentication level on your RubyGems account by August 15"

Reply:
Following this same spirit, I realized the heading of the email needed to be revised as well ([screenshot](https://screenshot.click/2022-24-08-31u2d-hdp9h.jpg)). So, I've gone ahead and updated that copy as well.